src/pages/MembersPage/MembersTable.tsx

```diff
@@ -931,38 +933,45 @@ export const MembersTable = ({
         perPage={perPage}
         sortByColumn={(index: number) => {
           if (index === 0)
-            return (u: ICircleUser) => u.profile.name.toLowerCase();
-          if (index === 1) return (u: ICircleUser) => u.address.toLowerCase();
-          if (index === 2) return (u: ICircleUser) => u.non_giver;
-          if (index === 3) return (u: ICircleUser) => u.non_receiver;
-          if (index === 4) return (u: ICircleUser) => u.fixed_payment_amount;
-          if (index === 6) return (u: ICircleUser) => u.isCircleAdmin;
+            return (u: QueryUser) => u.profile.name.toLowerCase();
+          if (index === 1) return (u: QueryUser) => u.address.toLowerCase();
+          if (index === 2) return (u: QueryUser) => u.non_giver;
+          if (index === 3) return (u: QueryUser) => u.non_receiver;
+          if (index === 4)
+            return (u: QueryUser) => u.user_private?.fixed_payment_amount;
+          if (index === 6) return (u: QueryUser) => isUserAdmin(u);
           if (index === 7)
-            return (u: ICircleUser) =>
+            return (u: QueryUser) =>
               !u.non_giver ? u.starting_tokens - u.give_token_remaining : -1;
           if (index === 8)
-            return (u: ICircleUser) =>
+            return (u: QueryUser) =>
               !!u.fixed_non_receiver || !!u.non_receiver
                 ? -1
                 : u.give_token_received;
-          return (u: ICircleUser) => u.profile.name.toLowerCase();
+          return (u: QueryUser) => u.profile.name.toLowerCase();
         }}
       >
         {member => (
           <MemberRow
-            isAdmin={me.isCircleAdmin}
             key={member.id}
             user={member}
             fixedPaymentToken={circle.fixed_payment_token_type}
             fixedPayment={fixedPayment}
-            tokenName={circle.tokenName}
+            token_name={circle.token_name}
             myUser={me}
             setDeleteUserDialog={setDeleteUserDialog}
-            setLeaveCircleDialog={setLeaveCircleDialog}
+            showLeaveModal={() => setShowLeave(true)}
             circleId={circle.id}
           />
         )}
       </MemberTable>
+      <LeaveCircleModal
```

Comment:
moving this modal from MembersPage/index wasn't strictly necessary for this work; it just seemed opportune to refactor to reduce the amount of prop-drilling